io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -84,6 +84,20 @@ def setUp(self):
         self.stripe = [64, 128, 256, 512, 1024]
         self.state = ['start', 'stop', 'start', 'pause', 'resume']
         self.list_test = ['online_offline', 'rebuild']
+        cmd = "%s /c%d set jbod=on" % (self.tool, self.controller)
+        if process.system(cmd, shell=True, ignore_status=True) != 0:
```

Comment:
We have a separate function `self.check_pass`. Can you use that please.

Reply:
@narasimhan-v  no these are pre-requisuite runs to bring the system in runnable state.  The output of this are not important to exit the tests which are handled via self.check_pass.

However let me know if self.check_pass can be extended to include informational messages only?  that can be done.

Reply:
@manvanthar I see `check_pass(self, cmd, errmsg)`  has the option to pass the error message.. so all `if process.system(` conditions here can be replaced by check_pass call

Reply:
@abdhaleegit i already responded up and explained on what can be done?.   Are you guys ok with the changes to check_pass